sdk/storage/azure-mgmt-storage/tests.yml

```diff
@@ -0,0 +1,16 @@
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      AllocateResourceGroup: 'false'
```

Comment:
is this line needed?

Reply:
Eventually we are phasing out support for `false`. You can try removing it and seeing if the deployment+tests still work.

Reply:
will try it out, thanks!